src/metabase/pulse/render.clj

```diff
@@ -125,53 +139,52 @@
 - content (a hiccup form suitable for rending on rich clients or rendering into an image)
 - render/text : raw text suitable for substituting on clients when text is preferable. (Currently slack uses this for
   scalar results where text is preferable to an image of a div of a single result."
-  [render-type timezone-id :- (s/maybe s/Str) card results]
+  [render-type timezone-id :- (s/maybe s/Str) card dashcard results]
   (let [{title :content, title-attachments :attachments} (make-title-if-needed render-type card)
+        {description :content}                           (make-description-if-needed dashcard)
         {pulse-body       :content
          body-attachments :attachments
          text             :render/text}                  (render-pulse-card-body render-type timezone-id card results)]
     (cond-> {:attachments (merge title-attachments body-attachments)
-             :content     [:a {:href   (card-href card)
-                               :target "_blank"
-                               :style  (style/style
-                                        (style/section-style)
-                                        {:margin          :16px
-                                         :margin-bottom   :16px
-                                         :display         :block
-                                         :text-decoration :none})}
-                           title
-                           pulse-body]}
+             :content [:p
+                       ;; Provide a horizontal scrollbar for tables that overflow container width.
+                       ;; Surrounding <p> element prevents buggy behavior when dragging scrollbar.
+                       [:div {:style (style/style {:overflow-x :auto})}
+                        [:a {:href        (card-href card)
+                             :target      "_blank"
```

Comment:
it's good practice to add `rel="noopener noreferrer"` as an attribute to an anchor tag whenever using `target="_blank"`

https://web.dev/external-anchors-use-rel-noopener/